src/components/EmployeeList.jsx

```diff
@@ -0,0 +1,104 @@
+import axios from "axios";
+import "../styles/dashboard.css";
+import { toast } from "react-toastify";
+import { useEffect, useState } from "react";
+import { Input, Dropdown, Button } from "antd";
+const { Search } = Input;
+import { RegistrationForm } from "./RegistrationForm";
+import { UserOutlined, UserAddOutlined } from "@ant-design/icons";
+
+export const EmployeeList = () => {
+  const [employees, setEmployees] = useState([]);
+  const [showForm, setShowForm] = useState(false);
+  const [searchType, setSearchType] = useState("Everyone");
+
+  const getAllEmployees = () => {
+    axios
+      .get("http://localhost:8080/api/emp/getemployees")
+      .then((response) => {
+        setEmployees(response.data.message);
+      })
+      .catch(() => {
+        toast.error("Error fetching employees");
+      });
+  };
```

Comment:
place api calls in a separate folder under src named 'api'
`api/`
-> `auth/` for auth operations
-> `user/` for user related requests
etc